conftest.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2023 Justin Myers for Adafruit Industries
```

Comment:
What is the point of this file? Is it run within any workflow? Locally on a device? do you have a future plan for it?

Reply:
conftest is for tests to build fixtures to reduce boiler plate. I have more in the `request` one here: https://github.com/adafruit/Adafruit_CircuitPython_Requests/blob/main/tests/conftest.py. I realize I moved that on into the test directory, and will do the same here